src/xenserver/opt/xensource/www/XCP-ng-index.html

```diff
@@ -18,6 +20,7 @@
       }
     </script>
     <link rel="stylesheet" href="asset/jquery.modal.min.css" />
+    <link rel="stylesheet" href="asset/fonts/icons.css" type="text/css" />
     <link rel="stylesheet" href="asset/flexboxgrid.min.css" type="text/css" />
     <link rel="stylesheet" href="asset/fontawesome-pro-5.8.1-web/css/all.css" />
```

Comment:
should not be necessary  anymore

Reply:
and this would allow to remove the whole fontawesome-pro-... directory

Reply:
All fontawesome stuff has been removed